code/game/machinery/Sleeper.dm

```diff
@@ -309,6 +316,31 @@
 				if(usr)
 					log_combat(usr,occupant, "injected morphine into", addition = "via [src]")
 				. = TRUE
+		if("bruteChem")
+			// SAFETY LOCKS DISENGAGED, OVERDOSE ENGAGED
+			mob_occupant.reagents.add_reagent(/datum/reagent/medicine/bicaridine, 5)
+			if(usr)
+				log_combat(usr,occupant, "injected 5u bicaridine into", addition = "via [src]")
+			. = TRUE
+		if("burnChem")
+			// SAFETY LOCKS DISENGAGED, OVERDOSE ENGAGED
+			mob_occupant.reagents.add_reagent(/datum/reagent/medicine/kelotane, 5)
+			if(usr)
+				log_combat(usr,occupant, "injected 5u kelotane into", addition = "via [src]")
+			. = TRUE
+		if("oxyChem")
+			// SAFETY LOCKS DISENGAGED, OVERDOSE ENGAGED
+			mob_occupant.reagents.add_reagent(/datum/reagent/medicine/salbutamol, 5)
+			if(usr)
+				log_combat(usr,occupant, "injected 5u salbutamol into", addition = "via [src]")
+			. = TRUE
+		if("radChem")
+			// SAFETY LOCKS DISENGAGED, OVERDOSE ENGAGED oh pentetic acid has no od :c
+			mob_occupant.reagents.add_reagent(/datum/reagent/medicine/pen_acid, 5)
+			if(usr)
+				log_combat(usr,occupant, "injected 5u pentetic acid into", addition = "via [src]")
+			. = TRUE
```

Comment:
This is ass, can it not be a "ChemHeal" that makes use of ``params``?

Reply:
Can do